docs/docs/cmd/entra/m365group/m365group-user-remove.mdx

```diff
@@ -37,19 +49,25 @@ You can remove users from a Microsoft 365 Group or Microsoft Teams team if you a
 Removes user from the specified Microsoft 365 Group.
 
 ```sh
-m365 entra m365group user remove --groupId '00000000-0000-0000-0000-000000000000' --userName 'anne.matthews@contoso.onmicrosoft.com'
+m365 entra m365group user remove --groupId '5b8e4cb1-ea40-484b-a94e-02a4313fefb4' --userName 'anne.matthews@contoso.onmicrosoft.com'
+```
+
+Removes user from the specified Microsoft 365 Team specified by id without confirmation.
+
+```sh
+m365 entra m365group user remove --teamId '5b8e4cb1-ea40-484b-a94e-02a4313fefb4' --userName 'anne.matthews@contoso.onmicrosoft.com' --force
```

Comment:
@MathijsVerbeeck did you commit all the changes? I think I we still have examples that use deprecated option which I would avoid